Cargo.lock

```diff
@@ -629,7 +662,7 @@ version = "0.8.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "17367f0cc86f2d25802b2c26ee58a7b23faeccf78a396094c13dced0d0182526"
 dependencies = [
- "phf_shared",
+ "phf_shared 0.8.0",
```

Comment:
Unlucky we got mulitple versions now...

Reply:
Yeah, at least overall it seems like there was a bit of a reduction in the wasm file's size.